.github/workflows/weekly-resources-clean.yml

```diff
@@ -29,7 +33,7 @@ jobs:
         with:
           aws-access-key-id: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
-          aws-region: us-west-2
+          aws-region: ${{ matrix.region }}
 
       - name: Clean old aws resources
         run: go run tools/workflow/clean_aws_resources.go
```

Comment:
I think you still need the `AWS_REGION` environment variable for it to use it.

Reply:
https://docs.aws.amazon.com/sdk-for-go/api/aws/session/
Doesn't seem to have anchors, so I can't link directly to it, but check out the "Environment Variables" section.

Reply:
From what I am understanding thought, ```AWS_SDK_LOAD_CONFIG``` is turn on; therefore, the session will use the the shared config (~/.aws/config) and shared credentials (~/.aws/credentials) files. Moreover, matrix is in separate instances. So I don't see why we should use the ```AWS_REGION``` though